src/modules/reserve-overview/graphs/InterestRateModelGraphContainer.tsx

```diff
@@ -53,7 +54,7 @@ export const InterestRateModelGraphContainer = ({
               baseStableBorrowRate: reserve.baseStableBorrowRate,
               baseVariableBorrowRate:
                 reserve.symbol === 'DAI' && dsr != null
-                  ? dsr.multipliedBy(1e27).toString()
+                  ? new BigNumber(reserve.variableBorrowAPR).multipliedBy(1e27).toString()
```

Comment:
This is correct as the APR, but I had the DSR as an APY, so the text still says "APY" incorrectly.